### ISSUE TITLE
Issue with Optional Attributes with Defaults

### ISSUE THREAD

**Issue by mathewpeterson** [massdriver-cloud/airlock#9]
Hello!

I am having issues converting from Terraform inputs that uses an optional attribute with default:

```hcl
variable "withoptionaldefault" {
  type = object({
    foo = string
    fizz = optional(string, "fizz")
  })
  default = {"foo":"bar"}
}
```

When I run:

```bash
$ airlock terraform input .
```

I get the following error:
```
Error: :3,20-36: Invalid type specification; Optional attribute modifier expects only one argument: the attribute type.

```

Terraform Docs: https://developer.hashicorp.com/terraform/language/expressions/type-constraints#example-nested-structures-with-optional-attributes-and-defaults

**Comment by mclacore** [comment]
Thanks @mathewpeterson for letting us know! I'll work on fixing that this week

**Comment by mclacore** [comment]
hey @mathewpeterson this issue should now be resolved with [v0.0.4](https://github.com/massdriver-cloud/airlock/releases/tag/v0.0.4)

**Comment by mathewpeterson** [comment]
I can confirm this is fixed, thank you!
